nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/WEB-INF/partials/canvas/import-flow-version-dialog.jsp

```diff
@@ -17,6 +17,11 @@
 <%@ page contentType="text/html" pageEncoding="UTF-8" session="false" %>
 <div id="import-flow-version-dialog" layout="column" class="hidden large-dialog">
     <div class="dialog-content">
+        <div class="setting keep-parameter-context">
```

Comment:
On the further review, placing this checkbox at the top of the dialog does not seem like the best approach. As this is an optional feature that may not be commonly used, recommend moving this checkbox to the end of the dialog window.